.github/workflows/deploy-standalone-plugins.yml

```diff
@@ -12,88 +13,65 @@ on:
         type: boolean
         description: 'Debug mode (run without publishing).'
         default: false
+
 jobs:
-  release:
-    name: Prepare Deployment
+  pre-run:
+    name: Pre-run
     runs-on: ubuntu-latest
     outputs:
-      matrix: ${{ steps.set-matrix.outputs.matrix }}
+      matrix: ${{ steps.set-matrix.outputs.json }}
     steps:
-      - name: Checkout repository
+      - name: Checkout
         uses: actions/checkout@v3
-      - name: Setup Node.js (.nvmrc)
-        uses: actions/setup-node@v3
-        with:
-          node-version-file: '.nvmrc'
-          cache: npm
-      - name: Install npm dependencies
-        run: npm ci
-      - name: Get directory
-        id: get-plugin-directory
-        if: ${{ github.event_name == 'workflow_dispatch' }}
-        run: |
-          echo "directory=$(node ./bin/plugin/cli.js get-plugin-dir --slug=${{ inputs.slug }})" >> $GITHUB_OUTPUT
-      - name: Get plugin version
-        id: get-version
-        if: ${{ github.event_name == 'workflow_dispatch' }}
-        run: |
-          echo "version=$(node ./bin/plugin/cli.js get-plugin-version --slug=${{ inputs.slug }})" >> $GITHUB_OUTPUT
+
       - name: Set matrix
         id: set-matrix
         run: |
+          PLUGINS=$(jq -r '.plugins' plugins.json)
           if ${{ github.event_name == 'workflow_dispatch' }}; then
-            result=$(echo "${{ steps.get-version.outputs.version }}" | awk '/^(\*|[0-9]+(\.[0-9]+){0,2}(-[a-zA-Z0-9.]+)?)$/ {print "Matched"}')
-            if [[ -n "$result" ]]; then
-              # Set the manual input values in JSON format for use in the matrix.
-              echo "matrix={\"include\":[{\"slug\":\"${{ inputs.slug }}\",\"version\":\"${{ steps.get-version.outputs.version }}\",\"directory\":\"${{ steps.get-plugin-directory.outputs.directory }}\",\"dry-run\":\"${{ inputs.dry-run }}\"}]}" >> $GITHUB_OUTPUT
```

Comment:
Why the manual workflow is removed? https://github.com/WordPress/performance/actions/workflows/deploy-standalone-plugins.yml?query=branch%3Aupdate%2Fplugin-bundling

Reply:
@mukeshpanchal27 Still working on the workflow files. Will update you once done.

Reply:
@mukeshpanchal27 I have updated the workflow files. Please feel free to re-review them.

Reply:
And JFI, we are now using build manifest instead of using a CLI script to determine the plugin versions.